dist/svg.js

```diff
@@ -1740,15 +1738,11 @@ SVG.FX = SVG.invent({
           this.situation.loop = currentLoop
         } else {
           this.pos = 1
-          ignoreTime = false
         }
```

Comment:
I need to take a deeper look into this block. Looks like magic^^
Btw: There is still `if(this.pos < 0) this.pos = 0`. I thought you wanted to allow values below 0